src/Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.hpp

```diff
@@ -24,12 +24,36 @@ struct Packet;
 
 namespace detail {
 
+const double proton_mass_MeV = 938.272046;
+
 void draw_single_packet(gsl::not_null<double*> time,
                         gsl::not_null<std::array<double, 3>*> coord,
                         gsl::not_null<std::array<double, 3>*> momentum,
                         gsl::not_null<std::mt19937*> random_number_generator);
 }  // namespace detail
 
+
+// Contribution to the neutrino-matter coupling terms from
```

Comment:
Make this a doxygen comment with the first sentence as \brief and the rest as \details